pkgs/development/libraries/rsocket-cpp/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, fetchFromGitHub, cmake, boost, libevent, double-conversion, glog, lib, fmt_8
+, gflags, openssl, folly }:
+
+stdenv.mkDerivation rec {
+  pname = "rsocket-cpp";
+  version = "2021.08.30.00";
+
+  src = fetchFromGitHub {
+    owner = "rsocket";
+    repo = "rsocket-cpp";
+    rev = "v${version}";
+    sha256 = "sha256-cYMc4aLP7iJ5yZ5tEg8Iui/OEbIA8xAZPOQcP3fAC8k=";
+  };
+
+  patches = [
+    ./folly-override-fix.patch
+  ];
+
+  nativeBuildInputs = [ cmake ];
+
+  cmakeFlags = [
+    "-DBUILD_EXAMPLES=off"
+    "-DBUILD_TESTS=off"
+    "-Wno-dev"
+  ];
+
+  buildInputs = [
+    fmt_8
```

Comment:
Why does it depend on `fmt_8` while other packages depend on `fmt_9`? Would there be any conflict?

Reply:
Just a typo I made during rebasing. I'll update to fmt_9 and ensure it still builds and runs